src/main/mondrian/spi/impl/JdbcDialectImpl.java

```diff
@@ -331,26 +331,10 @@ public void quoteIdentifier(final String val, final StringBuilder buf) {
             return;
         }
 
-        int k = val.indexOf('.');
```

Comment:
This was purposely put in to handle "owner.table". and replacing that with "owner"."table". based on this commit:
https://github.com/pentaho/mondrian/commit/2930d1256605148dbc7e2641e7f0b8904b5d7efc#diff-26f86eb9f6e0b15e1c4af7f0457b0801

Do your test cases cover that ?